ITEM.tsv

```diff
@@ -5300,15 +5300,15 @@ ITEM_20150714_005299	An orb that holds the aura
 ITEM_20150714_005300	{memo X}A beed that can collect the aura. It can be taken out and put depending on use. 
 ITEM_20150714_005301	Wisdom Guiding Stone 
 ITEM_20150714_005302	A mysterious jewel that can lead to the correct path.
-ITEM_20150714_005303	A truth eye Fragment 
+ITEM_20150714_005303	Eyes of Truth Fragment 
 ITEM_20150714_005304	{memo X}Truth Eye's broken pieces. It is only available to collect all fragments. 
-ITEM_20150714_005305	Truth Eyes 
+ITEM_20150714_005305	Eyes of Truth
 ITEM_20150714_005306	{memo X}allows you to see the reality of the soul. Let's try to insubstantial monster. 
 ITEM_20150714_005307	Soul Crystal 
 ITEM_20150714_005308	{memo X}wandering pulling force can pull the soul. It is necessary to find a condensed wanted. 
 ITEM_20150714_005309	Left-click to use 
 ITEM_20150714_005310	{memo X}A memo written by Hones about a teacher's said. You can know how to release the device installed at Ritnis catacombs. Cab read 
-ITEM_20150714_005311	Contract Space Magic Stone 
+ITEM_20150714_005311	Spatial Magic Gem of Contract Fragment
```

Comment:
Spatial Demonic Stone Fragment of Contract
Contract's Spatial Demonic Stone Fragment

Spatial Magic Stone Fragment of Contract
Contract's Spatial Magic Stone

Spatial Magic Gem Fragment of Contract
Contract's Spatial Magic Gem